public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -30,62 +30,57 @@ import {
 } from '@elastic/eui';
 import { ReportDetailsComponent } from '../report_details/report_details';
 
-const created_date = new Date('April 20, 2020 20:32:12');
+export function ReportDefinitionDetails(props) {
+  const [reportDefinitionDetails, setReportDefinitionDetails] = useState({});
+  const reportDefinitionId = props.match['params']['reportDefinitionId'];
 
-const reportDefinitionDetailsMockMetadata = {
-  name: '[Logs] Web traffic',
-  description: '--',
-  created: created_date.toString(),
-  last_updated: '--',
-  source: 'dashboards/daily_sales',
-  time_period: 'Last 30 minutes',
-  file_format: 'PDF',
-  report_header: '--',
-  report_footer: '--',
-  trigger_type: 'Schedule',
-  schedule_details: '--',
-  alert_details: '--',
-  status: 'Active',
-  delivery_channels: ['Kibana reports'],
-  kibana_recipients: ['admin'],
-  email_recipients: 'user1@email.com',
-  email_subject: 'Latest web traffic report',
-  email_body:
-    'View report details %REPORT_DETAILS_URL% Download report file %FILE_DOWNLOAD_URL%',
-  include_report_as_attachment: true,
-};
+  const handleReportDefinitionDetails = (e) => {
+    setReportDefinitionDetails(e);
+  };
 
-interface ReportDefinitionDetailsRouteProps {
-  reportDefinitionId: string;
-  reportDefinitionDetailsMetadata: {
-    name: string;
-    description: string;
-    created: string;
-    last_updated: string;
-    source: string;
-    time_period: string;
-    file_format: string;
-    report_header: string;
-    report_footer: string;
-    trigger_type: string;
-    schedule_details: string;
-    alert_details: string;
-    status: string;
-    delivery_channels: string[];
-    kibana_recipients: string[];
-    email_recipients: string;
-    email_subject: string;
-    email_body: string;
-    include_report_as_attachment: boolean;
+  const getReportDefinitionDetailsMetadata = (data) => {
+    let reportDefinitionDetails = {
+      name: data['report_name'],
+      description: data['description'],
+      created: data['time_created'],
+      last_updated: data['last_updated'],
+      source: data['report_source'],
+      time_period:
+        data['trigger']['trigger_params']['schedule']['interval']['period'] +
+        data['trigger']['trigger_params']['schedule']['interval']['days'],
+      file_format: data['report_params']['report_format'],
```

Comment:
Be consist on the naming, I think mostly it is `camelCase` Use everywhere the same case.